partcad/src/partcad/part_factory_brep.py

```
@@ -0,0 +1,38 @@
+    def __init__(self, ctx, source_project, target_project, config):
+        with pc_logging.Action("InitBREP", target_project.name, config["name"]):
+            super().__init__(
+                ctx,
+                source_project,
+                target_project,
+                config,
+                extension=".brep",
+            )
+            self._create(config)
+
+    async def instantiate(self, part):
```

Ruff (0.8.2)

Missing return type annotation for public function `instantiate`

(ANN201)